drivers/virtualbox/virtualbox.go

```diff
@@ -363,6 +363,14 @@ func (d *Driver) Create() error {
 		return err
 	}
 
+	cmd, err = d.GetSSHCommand(fmt.Sprintf("sudo hostname %s", d.MachineName))
```

Comment:
I don't believe this will persist reboots will it (need to update `/etc/hostname` at least).